README.md

```diff
@@ -20,13 +20,30 @@ Project DeepSpeech is an open source Speech-To-Text engine that uses a model tra
 * [TensorFlow 1.0 or above](https://www.tensorflow.org/install/)
 * [SciPy](https://scipy.org/install.html)
 * [PyXDG](https://pypi.python.org/pypi/pyxdg)
-* [python_speech_features](https://pypi.python.org/pypi/python_speech_features)
 * [python sox](https://pypi.python.org/pypi/sox)
 * [pandas](https://pypi.python.org/pypi/pandas)
+* [DeepSpeech native client libraries](https://tools.taskcluster.net/index/artifacts/#project.deepspeech.deepspeech.native_client.master/)
 
 
 ## Install
 
+### Installing pre-built DeepSpeech Python bindings
+
+Pre-built binaries can be found on [TaskCluster](https://tools.taskcluster.net/index/artifacts/#project.deepspeech.deepspeech.native_client.master/project.deepspeech.deepspeech.native_client.master.cpu). You'll need to download `native_client.xz` and the appropriate Python wheel package.
+
+First, the library files contained in `native_client.xz` need to be installed within the system library path.
```

Comment:
Well, that's what I mean by the system library path, but as you imply - better to talk in more generic terms I guess - I suppose on Mac this would end up being META_LD_LIBRARY_PATH, or whatever it is?